python/benchmark/gen_data_distributed.py

```diff
@@ -785,7 +802,6 @@ def make_sparse_regression_udf(
                     random_state=generator,
                     format="csr",
                     dtype=dtype,
-                    data_rvs=np.random.randn,
```

Comment:
Curious why this was deleted?

Reply:
For the function to reuse generator which was initialized with random_state. 
Keeping it here will instruct the function to use np.random.randn, which creates non-determinism and it was not initialized with random_state.

Reply:
Does the distribution change from normal to uniform 0,1?  To keep distribution could an initialized generator function be passed in here?

Reply:
Good catch. 

Just added back the data_rvs per np.random had been revised to be initialized with the partition seed.